data/overlayfiles/rancher-deployer/cnab/app/charts/csp-rancher-usage-operator/100.0.0+up0.1.0/templates/rbac.yaml

```diff
@@ -86,6 +86,20 @@ rules:
   - {{ template "csp-usage-operator.cspBillingAdapterConfigMap"  }}
   verbs:
   - get
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ .Chart.Name }}-archive-reader-role
```

Comment:
Do we really need two separate roles for this? Can't we just add the resource to the configmap-reader-role?